README.md

```diff
@@ -110,10 +110,12 @@ If `auth_type` is `Basic`, this must be the username and password, separated by
 If `auth_type` is `Bearer`, this is the token used for token bearer authorization, so requests must carry an `Authorization: Bearer <token>` header.
 
 
-### `markdown`
-This setting takes a boolean and specifies if the message should be sent as Markdown or as plaintext.
-If `false` (the default), the message will be sent as plaintext.
-Otherwise it will be sent as Markdown.
+### `markdown` and `html`
+These are two seperate config settings that take a boolean, and control how the message is sent.
+* If `markdown` is `true` and `html` is `true`, message will be sent as markdown.
+* If `markdown` is `false` and `html` is true, message will be sent as html.
+* If `markdown` is `true` and `html` is true, message will be sent as markdown (and a warning logged as markdown is the fallback in this case)
+* If `markdown` is `false` and `html` is false, message will be sent as plaintext.
```

Comment:
This gets a bit complicated with multiple booleans determining the message format, can you combine these options into one that takes a string for `markdown`, `plaintext` and `html`?

Reply:
yeah, i was just trying to keep backwards compatiblity its def more confusing than it needs to be

Reply:
Ah I see, you can keep backwards compatibility by adding a few lines to the `Config.do_update()` method, that check for an existing `markdown` option and then set the new option accordingly to `markdown` or `plaintext`.